openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java

```diff
@@ -559,6 +559,27 @@ void insert(
             + "ORDER BY extension")
     List<ExtensionRecord> getExtensions(@Bind("id") String id, @Bind("extensionPrefix") String extensionPrefix);
 
+    @RegisterRowMapper(ExtensionMapper.class)
+    @SqlQuery("SELECT extension, json FROM entity_extension WHERE id = :id AND extension LIKE :extension")
```

Comment:
@mohityadav766 any reason to add these, why not just pass it as field?

Reply:
need it in direct list apis, existing method has prefix match only